code/modules/mob/abstract/new_player/new_player.dm

```diff
@@ -5,6 +5,7 @@
 	var/spawning = 0//Referenced when you want to delete the new_player later on in the code.
 	var/totalPlayers = 0		 //Player counts for the Lobby tab
 	var/totalPlayersReady = 0
+	var/datum/late_choices/late_choices_ui = null
```

Comment:
clean this up in destroy

Reply:
Do you mean just setting the reference to null? I didn't bother since the UI gets closed automatically - transferring the player to a new mob closes all the UIs automatically, and that cleans it up. But I guess removing the reference there wouldn't hurt.